android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/navigation/NavigationMenuConfig.kt

```diff
@@ -53,5 +57,28 @@ data class ImageConfig(
   }
 }
 
+@Serializable
+data class ImageConfiguration(
+  val id: String,
+  override val resourceType: String,
+  val contentType: String,
+  val data: String,
+) : Configuration()
```

Comment:
I understand why this was needed. However with the fix on image generation via FHIR web it's unnecessary. The double decoding is not needed anymore, actually, this data class is a duplication of Binary resource so either way we do not need to create a new custom model.

Reply:
Noted @ellykits 
will revert the implementation to only decode once.